nativescript-imagepicker/images.ios.ts

```diff
@@ -17,3 +22,12 @@ export function done(args) {
 
     page.bindingContext.imagePicker.done();
 }
+
+function iOSProperty(_this, property) {
```

Comment:
Yes I know where it exists; I even have a blog article on it.  :-)
However, since plugins _should_ work with TNS <2.3; the "getter" isn't around until 2.3...    So being a plugin author, I tend to safe guard my code for at least a few versions.. ;-D